tbdex/offering/offering.go

```diff
@@ -149,10 +150,13 @@ func (o *Offering) Verify() error {
 		return fmt.Errorf("failed to verify Offering signature: %w", err)
 	}
 
-	// TODO once implemented in web5-go
-	// if jws.SignerDID != o.From {
-	// 	return errors.New("SignerDID: %w does not equal Offering.From: %w", jws.SignerDID, o.From)
-	// }
+	decoded, err := jws.Decode(o.Signature)
```

Comment:
Same as for RFQ, 
`decoded, err := tbdex.VerifySignature(o, o.Signature)`
above and you don't need to call `jws.Decode(...)` again